pegasus/sites.v3/code.org/views/stats_homepage.haml

```diff
@@ -8,20 +8,37 @@
   top_statistic_number = student_accounts_percent
   top_statistic_phrase = "of US students have accounts on Code.org"
 
-  statistics = {
-    female_students => "of our students are female",
-    teacher_users => "teachers use Code.org",
-    world_students_hoc_percent => "of the world's students have tried the Hour of Code",
-    states_policy_change => "states changed policy to support computer science"
-  }
+  statistics = [
+    {
+      number: female_students,
+      phrase: "of our students are female",
+      link: '/diversity'
+    },
+    {
+      number: teacher_users,
+      phrase: "teachers use Code.org",
+      link: CDO.studio_url('/courses?view=teacher')
+    },
+    {
+      number: world_students_hoc_percent,
+      phrase: "of the world's students have tried the Hour of Code",
+      link: 'https://hourofcode.com'
```

Comment:
I think `CDO.hourofcode_hostname` works here

Reply:
`CDO.hourofcode_hostname` => http://localhost:3000/localhost.hourofcode.com

Reply:
ah yes, the `_hostname` functions work a bit differently to the `_url` functions in https://github.com/code-dot-org/code-dot-org/blob/staging/deployment.rb.  Maybe we'd benefit from an `hourofcode_url` function added to that file.